client/pages/events/index.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
+import { EventViewToggle } from '../../components/events'
 import { useAppContext } from '../../context'
-import { EventList } from '../../components/events'
 
 export default function Events() {
   const { events } = useAppContext()
-
-  return <EventList events={events} />
+  const filteredEvents = Array.from(events.values())
```

Comment:
The name `filteredEvents` seems like a misnomer here?

Reply:
The intention was to tie in filters with the events passed in to the list view (both the calender and list event view depend on the events subject to any filters).